android/src/main/java/com/theweflex/react/WeChatModule.java

```diff
@@ -160,7 +164,7 @@ public void shareToTimeline(ReadableMap data, Callback callback){
     }
```

Comment:
Hi, @yorkie , We can see that, The code from Tencent official SDK use "WXSceneTimeline" to represent "朋友圈".
And I think we should keep our API compatible with previous version instead of breaking it.